corehq/apps/domain/views.py

```diff
@@ -657,7 +659,12 @@ def plan(self):
                     next_subscription.update({
                         'exists': True,
                         'date_start': subscription.next_subscription.date_start.strftime("%d %B %Y"),
+                        'name': subscription.next_subscription.plan_version.plan.name,
+                        'price': self.get_product_summary(subscription.next_subscription.plan_version,
+                                                          self.account,
+                                                          subscription)[0]['monthly_fee'],
```

Comment:
"we think there is never an instance when `get_product_summary()` returns more than 1 element" - could always put an `assert` for this if you wanted to fail hard in that scenario